pyface/color.py

```diff
@@ -25,7 +25,8 @@
     Bool, HasStrictTraits, Property, Range, Tuple, cached_property
 )
 
-from pyface.util.color_helpers import channels_to_ints, ints_to_channels, is_dark
+from pyface.util.color_helpers import channels_to_ints, is_dark
+from pyface.util.color_helpers import ints_to_channels  # noqa: F401
```

Comment:
I see imports of `ints_to_channels` from `pyface.color`... Are those imports technically wrong?

Reply:
I mean i can understand why devs would prefer doing `from pyface.color import ints_to_channels, channels_to_ints` instead of `pyface.util.color_helpers`.

Explicitly mentioning that along with the `#noqa` would probably be better.

Reply:
Having fixed a mess of transitive imports in the past, I get uncomfortable when I see this.

Reply:
@corranwebster did you intentionally want to re-export `channels_to_ints` and `ints_to_channels` via `pyface.color`?

Reply:
im merging this for now - and open a separate issue to discuss whether or not we should recommend using `channels_to_ints` and `ints_to_channels` from `pyface.color`